packages/pxweb2/src/app/components/FilterSidebar/FilterSidebar.tsx

```diff
@@ -0,0 +1,71 @@
+import cl from 'clsx';
+
+import { type State, type Filter } from '../../pages/StartPage/tableTypes';
+import styles from './FilterSidebar.module.scss';
+
+import { Checkbox } from '@pxweb2/pxweb2-ui';
+interface FilterProps {
+  state: State;
+  handleAddFilter: (filter: Filter[]) => void;
+  handleRemoveFilter: (filter: Filter) => void;
+  handleResetFilter: () => void;
+}
+
+export const FilterSidebar: React.FC<FilterProps> = ({
+  state,
+  handleAddFilter,
+  handleRemoveFilter,
+  handleResetFilter,
+}) => {
+  return (
+    <div className={styles.sideBar}>
+      <h2 className={cl(styles['heading-small'])}>Filter</h2>
+      <div>
+        <button
+          onClick={() =>
+            handleAddFilter([{ type: 'variableName', value: 'region' }])
+          }
+        >
+          <span className={cl(styles['label-medium'])}>
+            Filter: Only tables with variableName "region"
+          </span>
+        </button>
+      </div>
+      <div>
+        <button onClick={handleResetFilter}>
+          <span className={cl(styles['label-medium'])}>Filter: Reset!</span>
+        </button>
+      </div>
+      <div>
+        <div className={cl(styles['heading-medium'])}>Filter</div>
+        <div className={cl(styles['heading-small'])}>Tidsintervall</div>
+        <ul className={styles.filterList}>
+          {Array.from(state.availableFilters)
```

Comment:
Consider doing this operation with maps etc outside of the JSX, so that its more readable

Reply:
Moved it out into a separate function.